morph/react/get-dependencies.js

```diff
@@ -29,7 +29,8 @@ export default (state, getImport) => {
     }
   }
 
-  const dependencies = []
+  const dependencies = [`import React from 'react'`]
```

Comment:
do we need this on native now as well?

Reply:
It was always there, I just moved it in the deps now so we can sort them properly :). RN is a target like DOM, they share the react package though